rhino/src/main/java/org/mozilla/javascript/regexp/NativeRegExp.java

```diff
@@ -780,6 +1029,26 @@ private static int getDecimalValue(char c, CompilerState state, String overflowM
         return value;
     }
 
+    private static RENode reverseNodeList(RENode head) {
```

Comment:
I'm just glad that someone got to write code to "reverse a linked list" in real-world code! ;-)